tui/menu_button.go

```diff
@@ -142,11 +142,16 @@ func (mb *MenuButton) Draw() {
 	clui.PushAttributes()
 
 	itemValue := fmt.Sprintf("%c %s", statusSymbol[mb.status], mb.itemValue)
-	shift, itemText := clui.AlignColorizedText(itemValue, w, mb.Align())
+
+	if len(itemValue) >= w {
+		itemValue = fmt.Sprintf("%s...", clui.CutText(itemValue, w-7))
```

Comment:
Why the magic number 7?
Testing this by enabling all current bundles results in only one black space after the "..."; given the magic number 7 and then putting in three '.'s plus on space -- there seem to be 4 characters still missing?
Can we make this a calculation?

Reply:
done